.circleci/config.yml

```diff
@@ -67,6 +67,18 @@ jobs:
       - run: docker build . -f ./build/Dockerfile-rhel -t $OAO_IMAGE_RHCC:$CIRCLE_TAG
       - run: docker push $OAO_IMAGE_RHCC:$CIRCLE_TAG
 
+  deploy-release-snapshot:
+    <<: *defaults
+    steps:
+      - setup_remote_docker
+      - run: |
+          mkdir -p ~/.docker && chmod 0700 ~/.docker
+          touch ~/.docker/config.json && chmod 0600 ~/.docker/config.json
+          base64 -d >~/.docker/config.json <<<"$OAO_DOCKER_AUTH"
+      - attach_workspace:
+          at: .
+      - run: docker build . -f ./build/Dockerfile -t $OAO_IMAGE:$CIRCLE_TAG
```

Comment:
We'd like to have a different tag for these and not just "release-0.2" Docker tags since it may confuse the customers. Something like "snapshot-$CIRCLE_TAG" perhaps.

Reply:
As written above, I'd suggest:
`v0.3-snapshot` to conform with our release tags.

Reply:
Fine with me.